examples/CSharp/Tutorial 1.6 - Hello Gamepad/Program.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using Silk.NET.Input;
+using Silk.NET.Maths;
+using Silk.NET.Windowing;
+
+namespace Tutorial_1._6___Hello_Gamepad
+{
+    class Program
+    {
+        private static IInputContext inputContext;
+        private static IWindow window;
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+            CreateWindow();
+            window.Run();
+        }
+
+        private static void CreateWindow()
+        {
+            //Create a window.
+            var options = WindowOptions.Default;
+            options.Size = new Vector2D<int>(800, 600);
+            options.Title = "Tutorial 1.6 - Hello Gamepad";
+
+            window = Window.Create(options);
+
+            //Assign events.
+            window.Load += OnLoad;
+            window.Update += OnUpdate;
+            window.Render += OnRender;
+        }
+        private static void OnLoad()
+        {
+            //Set-up input context.
+            inputContext = window.CreateInput();
+            inputContext.ConnectionChanged += ConnectionChanged;
+
+            for (int i = 0; i < inputContext.Keyboards.Count; i++)
+            {
+                inputContext.Keyboards[i].KeyDown += KeyDown;
+            }
+
+            //if a controller is plugged in before the application is launched, ConnectionChanged may or may not fire depending on the underlying driver used.
```

Comment:
This behavior is inconsistent between glfw and sdl. Registering gamepads OnLoad and on ConnectionChanged seems to capture both scenarios well enough for a basic tutorial